notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -551,7 +551,7 @@
     "xtalk_overlaps = {}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                              f'Approximate Signal Loss with
{-XTALK_FR} to {XTALK_FR} mHz X-Talk Filter': [f'{loss:.1%}' for loss in frf_losses.values()],
</pre><p>We can remove the word "Approximate"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>